oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/ordfm/RDFTypes.java

```diff
@@ -0,0 +1,179 @@
+package org.eclipse.lyo.oslc4j.provider.jena.ordfm;
+
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.ClassInfo;
+import io.github.classgraph.ClassInfoList;
+import io.github.classgraph.ScanResult;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.vocabulary.RDF;
+import org.eclipse.lyo.oslc4j.core.annotation.OslcResourceShape;
+import org.eclipse.lyo.oslc4j.core.model.TypeFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * 
+ * @author rherrera
+ */
+public class RDFTypes {
```

Comment:
Given the contents of the package, I would rather rename it to `OslcResourceUtil`

Reply:
Is it ok if we rename it to "Resources" instead? Once I read a paper saying the use of "Util" in a class name was not a good practice and the author suggested the use of the plural instead, he listed some Java core classes as example: Collections, Arrays, Objects. I chose the name RDFTypes (and inside the package ordfm) because I was thinking we could later move this class out (along with JMH) to an independent jar for RDF-to-Class marshaling/unmarshalling; some times you need to do these conversions outside a JAX-RS environment and it's weird you have to include a providers dependency.

Reply:
How about `ResourcePackages`? It only deals with resources in the packages and does not deal with other OSLC Resource aspects eg handled in the JMH.